osh.spec

```diff
@@ -188,6 +188,16 @@ rm -rf %{buildroot}%{python3_sitelib}/scripts
 # install example httpd config
 install -D {osh/hub,%{buildroot}%{_sysconfdir}/httpd/conf.d}/osh-hub-httpd.conf
 
+# install -D {osh/hub,%{buildroot}%{_sysconfdir}/osh/hub}/settings.py
+# install -D {osh/hub,%{buildroot}%{_sysconfdir}/osh/hub}/settings_local.ci.py
+
+mv %{buildroot}%{python3_sitelib}/osh/hub/settings.py %{buildroot}%{_sysconfdir}/osh/hub
+mv %{buildroot}%{python3_sitelib}/osh/hub/settings_local.ci.py %{buildroot}%{_sysconfdir}/osh/hub/
```

Comment:
On a second look, we may not need this file at all in the rpm package, as the CI uses it [directly](https://github.com/openscanhub/openscanhub/blob/bd8475834908a317f552cb9321e24b2da46721b1/tests/ci.sh#L12) from git repository.